test/infrastructure/docker/Makefile

```diff
@@ -229,11 +244,8 @@ clean-release: ## Remove the release folder
 .PHONY: verify
 verify:
 	./hack/verify-boilerplate.sh
-	./hack/verify-doctoc.sh
-	./hack/verify-shellcheck.sh
 	$(MAKE) verify-modules
 	$(MAKE) verify-gen
-	$(MAKE) verify-modules
```

Comment:
Shouldn't we keep verify-modules given that we're removing the check in the root Makefile?

Reply:
Why do we need to verify twice?

Reply:
From the top Makefile diff:
```
 ## --------------------------------------
 ## Docker
@@ -357,11 +356,10 @@ verify:
        ./hack/verify-shellcheck.sh
        $(MAKE) verify-modules
        $(MAKE) verify-gen
-       $(MAKE) verify-modules
```

Reply:
The other line diff from the top Makefile removes `test/infrastructure/docker/go.sum` check

Reply:
Will fix

Reply:
Oh wait, yeah, I stand by my original statement. This diff is removing a second call to `verify-modules`.

Reply:
ah ok, makes sense now :)